deploy/generate.go

```diff
@@ -1,8 +0,0 @@
-package deploy
-
-// Copyright (c) Microsoft Corporation.
-// Licensed under the Apache License 2.0.
-
-//go:generate go run ../hack/gendeploy
```

Comment:
what happened to gendeploy here ? I could not find it somewhere else and we might need it to be run during make generate right? I think that passed through the net. 

Could we maybe put it in `pkg/deploy` (that one might require a slight refactor of `hack/gendeploy`) or in `pkg/deploy/assets` (after it has been renamed from `staticassets`)

Reply:
Good catch!  I'll get it added back.